docs/release-notes/2.44.0.md

```diff
@@ -0,0 +1,33 @@
+# Percona Monitoring and Management 2.44.0
+
+| **Release date** | December 9th, 2024                                                                                     |
+| -----------------| ----------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](../quickstart/index.md) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them, and execute database management operations regardless of whether your databases are located on-premises or in the cloud.
+
+## Improved PostgreSQL 17 Metrics Collection
+
+PMM 2.44.0 strengthens monitoring capabilities for PostgreSQL 17, building on the recent support for Query Analytics introduced in PMM 2.43.2. 
+With this update, we've revised queries tailored to PostgreSQL 17's updated schema to ensure that metrics are accurately collected and displayed in the **PostgreSQL Overview** dashboard.
+
+Columns like `checkpoints_timed` in the `stat_bgwriter` collector are now also correctly recognized.
+
+### [Tech Preview] Support for PSMDB  and Community MongoDB 8.0
+
+The latest version of MongoDB, along with [Percona Server for MongoDB 8.0](https://www.percona.com/software/mongodb/percona-server-for-mongodb), brings [numerous improvements and significant performance enhancements](https://www.mongodb.com/docs/manual/release-notes/8.0/).
+
+With this update, we are adding support for MongoDB 8 in PMM 2, including updates to `mongodb_exporter` to accommodate the revised metrics structure (e.g., `wiredTiger.concurrentTransactions` is now `queues.execution`).
+
+In addition, the MongoDB **Oplog Details** dashboard is adapted to support MongoDB 8.0's new oplog metrics, with updated panels for **Oplog Buffered Operations** and **Buffer Capacity**.
+
+![Updated Oplog Details dashboard](Oplog_Details.png)
+This enhancement requires PMM Agent version 2.43.1 or later. Keep in mind that some dashboard metrics may need further updates to fully support MongoDB 8.0's new format.
+
+## Fixed PBM metrics collection
+
+We've resolved an issue where Percona Backup for MongoDB (PBM) metrics were not being scraped by default and required the `--enable-all-collectors` flag.
+
+With this fix, PBM metrics are now automatically collected when MongoDB services are added to PMM, without requiring additional configuration.
```

Comment:
we also disabled pbm metrics collection for PMM Clients 2.43.0 and 2.43.1 as there were a memory leak. So now pbm metrics collection is supported only for PMM Clients starting 2.43.2